README.md

```diff
@@ -0,0 +1,49 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
```

Comment:
- Please, kindly update the project name.

Reply:
@zieeco Done